shared/clinical-code-sets/conditions/genital-tract-infections/1/README.md

```diff
@@ -0,0 +1,76 @@
+# Genital tract infections
+
+Infections of the male or female upper genital tract. Including pelvic inflammatory disease, salpingitis, endometritis, oophritis, cervicitis, epididymitis, orchitis, prostatitis. Excluding superficial infections e.g. vulvovaginitis, balanitis.
+
+This code set was created from a Readv2 code set developed from the University of Bristol:
+
+Jessica Watson (2018): CPRD codes: infections and autoimmune diseases. https://doi.org/10.5523/bris.2954m5h0ync672u8yzx16xxj7l
+
+It was then mapped to EMIS, CTV3 and SNOMED, and supplemented with codes from a traversal of the SNOMED hierarchy.
+
+## Prevalence log
+
+By examining the prevalence of codes (number of patients with the code in their record) broken down by clinical system, we can attempt to validate the clinical code sets and the reporting of the conditions. Here is a log for this code set. The prevalence range `7.3 - 8.5%` suggests that this code set is well defined.
+
+| Date       | Practice system | Population | Patients from ID | Patient from code |
+| ---------- | --------------- | ---------- | ---------------: | ----------------: |
+| 2024-07-09 | EMIS            | 2533608    |   215633 (8.51%) |    215616 (8.51%) |
+| 2024-07-09 | TPP             | 201974     |    14831 (7.34%) |     14831 (7.34%) |
+| 2024-07-09 | Vision          | 335755     |    26957 (8.03%) |     26902 (8.01%) |
+
+## Audit log
+
+- Find_missing_codes last run 2024-07-05
+
```

Comment:
is the text below meant to be in this README file?

Reply:
Good spot. Now removed.